src/styles/blocks/_catalog.scss

```
@@ -0,0 +1,125 @@
+@import "utils/variables";
+
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  max-width: 944px;
+  box-sizing: content-box;
+  &__card {
+    margin: 0 48px 46px 0;
+    padding: 32px 16px 16px;
+  }
+}
+
+.card {
+  max-width: 200px;
+  background-color: #fff;
```

You created variables for colors, but not for all 🥺 

```suggestion
  background-color: $c-white;
```